pkg/deploytest/localtransport.go

```diff
@@ -0,0 +1,24 @@
+package deploytest
+
+import (
+	"context"
+	"github.com/filecoin-project/mir/pkg/events"
+	"github.com/filecoin-project/mir/pkg/modules"
+	"github.com/filecoin-project/mir/pkg/pb/messagepb"
+	t "github.com/filecoin-project/mir/pkg/types"
+)
+
+type LocalTransport interface {
+	Link(source t.NodeID) Link
+}
+
+type Link interface {
```

Comment:
Why is it called Link?
We have the Transport interface with the similar functionality - https://github.com/filecoin-project/mir/blob/main/pkg/net/transport.go

Reply:
thanks, I changed it to `net.Transport` instead.
Also, modified `net.Transport` so that it extends `modules.ActiveModule` instead of repeating it.